Makefile

```diff
@@ -217,9 +220,15 @@ image-push-helm-multiarch:
 # Scorecard test image scaffold/build/push.
 .PHONY: image-build-scorecard-test image-push-scorecard-test image-push-scorecard-test-multiarch
 
+# Scorecard custom test image scaffold/build/push.
+.PHONY: image-build-custom-scorecard-tests image-push-custom-scorecard-tests image-push-custom-scorecard-tests-multiarch
+
 # Scorecard test kuttl image scaffold/build/push.
 .PHONY: image-build-scorecard-test-kuttl image-push-scorecard-test-kuttl image-push-scorecard-test-kuttl-multiarch
 
+image-build-custom-scorecard-tests:
+	./hack/image/build-custom-scorecard-tests-image.sh $(CUSTOM_SCORECARD_TESTS_BASE_IMAGE):dev
+
```

Comment:
IHMO we should not add more images since we have a lot already just for the tests. We could provide the code and doc how the users can build and push for their registry instead.

Reply:
@joelanford  Joe should probably decide if this example image lives inside this repo or outside in a separate repo (e.g. operator-sdk-samples).  I'm fine either way.  If its outside the repo then it will likely require on-going maintenance or some other external test to insure it does not get stale.